content/en/platform/corda/5.0/deploying-operating/observability/metrics.md

```diff
@@ -230,3 +235,89 @@ the maximum value, and the cumulative sum of values, respectively.
+
+#### Membership Worker
+
+The membership worker is responsible for the processing of application network functionality either on behalf of an MGM,
+a network member, or both. For an MGM, examples of this application functionality include:
+* handling incoming registration requests,
+* network management,
+* and ensuring the network participants are all in sync with the latest network data.
+
+For a network member, examples include:
+* registering with an MGM to join a network,
+* managing network data sent from the MGM,
+* and periodically syncing network data with the MGM.
+
+The timer metrics of the membership worker focus on the areas mentioned above. Specifically, the timer metrics cover:
+* each stage of registration on both the MGM and membership side,
+* general membership actions (which at the moment only include the distribution of network data by the MGM),
+* and the synchronisation of network data handling on both the MGM and member sides.
+
+These metrics are tagged with the name of the handler so that you can observe at a low-level exactly where time is spent
+across different processes. These handler names are tagged as the operation name. They are also tagged with the short
+hash ID and the group ID of the virtual node the operation is performed on behalf of in order to determine if certain
+virtual nodes or groups are taking longer than others to process.
+
+The membership worker also includes a single gauge metric which shows the size of the network member list held in memory
+at any point. It is useful to compare any changes in the performance returned by the timer metrics to the size of the
+member list at the time to see if timings could be impacted by a growing network. The gauge metrics are also tagged by
+virtual node short hash and group ID so that it is possible to see the overall size of the member lists held in memory
+but also view it per network of virtual node.
+
+| Metric | Type | Tags | Description |
+| :----------- | :----------- | :----------- | :----------- |
+| `corda_membership_actions_handler_time_seconds` | Timer | <ul><li>`operation_name`</li><li>`group`</li><li>`virtualnode`</li></ul> | The time spent on membership actions. The `operation_name` tag is the name of the operation that the metric is related to. For example, `DistributeMemberInfo`, `StartRegistration`, `QueryMemberInfo`, and so on. The `group` tag is the membership group ID of the virtual node performing an operation or being monitored. It can also appear as `not_applicable` when a membership group identifier is not accessible when the metric was collected. The `virtualnode` tag is the virtual node short hash of the virtual node performing an operation or being monitored. This can also appear as `not_applicable` when a virtual node identifier was not accessible when the metric was collected.  |
```

Comment:
The description here is mainly the tags description rather than the metrics description 
```
Measurement of time spent on membership actions. Currently this list is limit only to distribution of member info after a change to the network definition (group parameters / member list) but additional action may be added in the future. For now this would measure the time spent preparing the distribution packages sent out to members.
```

Would it be better to move the tag description to the intro since the descriptions are applicable to all metrics?